bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/init/StartupJob.java

```diff
@@ -279,11 +277,67 @@ private void onProjectCleaned(IProject project) {
                     return;
                 }
             } catch (CoreException e) {
-                // TODO Auto-generated catch block
                 Logger.log(IStatus.ERROR, "" + e);
                 Logger.log(IStatus.INFO, "" + e.getStackTrace());
             }        
-            //TODO UPLIFT Clean REPORTS??? 
+            // Delete the old logfile
+            CodeCheckerContext.getInstance().getCcProject(project).deleteMasterLogFile();
+        }
+    }
+
+    private static class ISaveCommandExecutionListener implements IExecutionListener {
+        private static AnalyzeJob analyzeJob;
+        private static PlistParseJob plistParseJob;
+        
+        @Override
+        public void notHandled(String commandId, NotHandledException exception) {
+            ;
+        }
+
+        @Override
+        public void postExecuteFailure(String commandId, ExecutionException exception) {
+            ;
+        }
+
+        @Override
+        public void postExecuteSuccess(String commandId, Object returnValue) {
+            if (IWorkbenchCommandConstants.FILE_SAVE.equals(commandId)) {
+                Logger.log(IStatus.INFO, "There was a save event!");
+                try {
+                    analyzeJob.schedule();
+                    plistParseJob.schedule();
+                } catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Catching 'Exception' is not allowed.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641620)